apps/shooting-clubs/acceptance/features/location-address.js

```diff
@@ -0,0 +1,220 @@
+'use strict';
+
+const steps = require('../../index');
```

Comment:
Don't think you need the index as it should automatically look for the index file